controllers/doguInstallManager.go

```diff
@@ -27,6 +26,8 @@ import (
 )
 
 const k8sDoguOperatorFieldManagerName = "k8s-dogu-operator"
+const k8sNginxIngressDoguName = "nginx-ingress"
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>const `k8sNginxIngressDoguName` is unused
<!-- __reviewdog__:Cg9iOTY0MTNmMDA1M2ZjZjESCmNoZWNrc3R5bGU= -->